### PR TITLE
Missing commands for demo data on docker docs

### PR DESCRIPTION
**Pull request by Mte90**

### CODE REVIEW

**Comment by akatsoulas** [comment]
Thanks @Mte90 for this PR! r+

File: docs/installation-docker.rst [hunk 1]
```diff
@@ -60,6 +60,18 @@ When you want to start contributing...
...
 
+#. Add demo users.
+
+    $ docker-compose run web ./manage.py loaddata demo_users
+
+#. Add demo functional areas.
+
+    $ docker-compose run web ./manage.py loaddata demo_functional_areas
+
+#. Add demo functional areas.
```
Comment (akatsoulas) [inline docs/installation-docker.rst]: Please change this to `Add demo events.`

**Comment by akatsoulas** [comment]
@Mte90 can you please merge these two commits into one?

**Comment by Mte90** [comment]
Done

**Comment by akatsoulas** [comment]
Thanks! r+ :rocket:

**Pull request merged**